compiler/noirc_evaluator/src/ssa/ir/function_inserter.rs

```diff
@@ -79,6 +79,12 @@ impl<'f> FunctionInserter<'f> {
         (instruction, self.function.dfg.get_instruction_call_stack_id(id))
     }
 
+    /// Get an instruction, map all its values, and replace it with the resolved instruction.
+    pub(crate) fn map_instruction_in_place(&mut self, id: InstructionId) {
+        let mut instruction = self.function.dfg[id].clone();
+        instruction.map_values_mut(|id| self.resolve(id));
+        self.function.dfg.set_instruction(id, instruction);
+    }
```

Comment:
```suggestion
    }
    
```
nit just to separate the `}` from the next comment